src/main/java/uk/gov/hmcts/reform/idam/web/UserController.java

```diff
@@ -144,7 +153,7 @@ public String selfRegister(
 
         Optional<Service> service;
 
-        if (StringUtils.isEmpty(clientId) || StringUtils.isEmpty(redirectUri)) {
+        if (isEmpty(clientId) || isEmpty(redirectUri)) {
```

Comment:
Hi, doesn't this `isEmpty`/`isBlank` thing apply here too?

Reply:
I rather leave this behaviour as it was.